src/pages/commits/ActiveCommits/CommitChartTooltip.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { taskStatusToCopy, mapTaskStatusToColor } from "constants/task";
+import { TaskStatus } from "types/task";
+import { ColorCount, getZeroCountStatus } from "./utils";
+
+const { gray } = uiColors;
+interface Props {
+  groupedTaskStats: ColorCount[];
+  trigger: React.ReactElement;
+}
+
+export const CommitChartTooltip: React.FC<Props> = ({
+  groupedTaskStats,
+  trigger,
+}) => {
+  const zeroCountStatus = getZeroCountStatus(groupedTaskStats);
+  return (
+    <Tooltip
+      usePortal={false}
+      align="right"
+      justify="middle"
+      popoverZIndex={1}
+      trigger={trigger}
+      triggerEvent="hover"
+    >
+      <TooltipContainer data-cy="commit-chart-tooltip" css={sharedCss}>
+        {groupedTaskStats.map(({ color, count, statuses, umbrellaStatus }) => (
+          <FlexColumnContainer key={color}>
+            <TotalCountContainer data-cy="current-status-count">
+              <Circle color={color} />
+              <StatusText
+                css={sharedCss}
+              >{`Total ${taskStatusToCopy[umbrellaStatus]}`}</StatusText>
+              <NumberText css={sharedCss}>{count}</NumberText>
+            </TotalCountContainer>
+            {umbrellaStatus === TaskStatus.Failed && (
+              <SubStatusText css={sharedCss}>{`(${statuses.join(
+                ", "
+              )})`}</SubStatusText>
+            )}
+          </FlexColumnContainer>
+        ))}
+        {zeroCountStatus.map((status) => (
+          <TotalCountContainer
+            opacity={0.4}
```

Comment:
Instead of passing in the opacity as a value lets declare this as a toggle-able prop and let emotion determine the style.
```suggestion
            active={false}
```

Reply:
you can name it something else.